spec/acceptance/basic_spec.rb

```diff
@@ -5,7 +5,8 @@
     let(:manifest) { 'include motd' }
   end
 
-  specify do
-    expect(file('/etc/motd')).to have_attributes(content: /Foreman project/)
```

Comment:
This is (according to rubocop-rspec maintainers) the preferred syntax. You can chain with `.and`:
```ruby
specify do
  expect(file('/etc/motd')).to have_attributes(content: /Foreman project/).and(have_attributes(content: /.../))
end

Reply:
This is (according to me) unreadable ;-)

But I can change it back if you want to. How does the look of the PR look like?

Reply:
How is this for readability?

```ruby
specify do
  expect(file('/etc/motd')).to be_file
    .and(have_attributes(content: /Foreman project/))
    .and(have_attributes(content: /.../))
end
```

The `have_attributes` is an implementation in core as a replacement for [rspec-its](https://github.com/rspec/rspec-its). I'll agree the `with()` custom matcher in rspec-puppet reads a bit easier, but we don't have that here.

Reply:
slightly better? updated. thanks.